app/assets/javascripts/base.js

```diff
@@ -3,7 +3,7 @@ $(document).ready(function() {
   $('body').tooltip({selector: "[data-toggle~='tooltip']", html: true});
 
   setTimeout(function() {
-    $(".alert").not('.alert-confirm').alert('close');
+    $(".alert").not('.alert-confirm, .scheduling-error').alert('close');
```

Comment:
This was the sneaky code causing the alert to disappear, sometimes very quickly.

Reply:
Interesting. Good find.